logdb/logdb_test.go

```diff
@@ -428,3 +437,317 @@ func TestLogDB_HasBlockID(t *testing.T) {
 	}
 	assert.True(t, has)
 }
+
+func CreateTempDBPath() (string, error) {
```

Comment:
Should it be private ?

```suggestion
func createTempDBPath() (string, error) {
```